drivers/tapdisk-metrics-stats.h

```diff
@@ -34,8 +34,6 @@
 
 #include <stdint.h>
 
-#define BT3_LOW_MEMORY_MODE 0x0000000000000001
```

Comment:
Is there also code to be removed where the stats are parsed?

Reply:
put the flags back